drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -431,18 +434,53 @@ function DragAndDropTemplates(configuration) {
                             {
                                 innerHTML: gettext("Close")
                             }
-                        ),
+                         ),
                         h(
                             'span.icon.fa.fa-times-circle',
                             {
                                 attributes: {
                                     'aria-hidden': true
                                 }
                             }
+                         )
+                    ]
+                 ),
+                h(
+                    ctx.last_action_correct ? 'div.popup-header-icon' : 'div.popup-header-icon.popup-header-icon-incorrect',
```

Comment:
Nit: for consistency, it would be nice if we also used `popup-header-icon-correct`class.